[2021-12-22 01:38:59,653] {processor.py:163} INFO - Started process (PID=40) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:38:59,673] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:38:59,680] {logging_mixin.py:109} INFO - [2021-12-22 01:38:59,677] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:38:59,770] {logging_mixin.py:109} INFO - [2021-12-22 01:38:59,761] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 42, in <module>
    bash_command='python /opt/airflow/resoures/test.py'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 143, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 547, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (call python sript) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:38:59,806] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:38:59,822] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.175 seconds
[2021-12-22 01:39:30,020] {processor.py:163} INFO - Started process (PID=70) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:39:30,022] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:39:30,024] {logging_mixin.py:109} INFO - [2021-12-22 01:39:30,024] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:39:30,046] {logging_mixin.py:109} INFO - [2021-12-22 01:39:30,043] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 42, in <module>
    bash_command='python /opt/airflow/resoures/test.py'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 143, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 547, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (call python sript) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:39:30,069] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:39:30,082] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.067 seconds
[2021-12-22 01:40:48,937] {processor.py:163} INFO - Started process (PID=39) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:40:48,949] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:40:48,958] {logging_mixin.py:109} INFO - [2021-12-22 01:40:48,958] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:40:49,013] {logging_mixin.py:109} INFO - [2021-12-22 01:40:49,010] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 42, in <module>
    bash_command='python /opt/airflow/resoures/test.py'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 143, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 547, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (call python sript) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:40:49,114] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:40:49,158] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.232 seconds
[2021-12-22 01:41:19,285] {processor.py:163} INFO - Started process (PID=64) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:41:19,287] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:41:19,288] {logging_mixin.py:109} INFO - [2021-12-22 01:41:19,288] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:41:19,307] {logging_mixin.py:109} INFO - [2021-12-22 01:41:19,304] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 42, in <module>
    bash_command='python /opt/airflow/resoures/test.py'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 143, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 547, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (call python sript) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:41:19,326] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:41:19,338] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.059 seconds
[2021-12-22 01:41:49,435] {processor.py:163} INFO - Started process (PID=89) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:41:49,438] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:41:49,439] {logging_mixin.py:109} INFO - [2021-12-22 01:41:49,439] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:41:49,453] {logging_mixin.py:109} INFO - [2021-12-22 01:41:49,450] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 42, in <module>
    bash_command='python /opt/airflow/resoures/test.py'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 143, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 547, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (call python sript) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:41:49,472] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:41:49,481] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.049 seconds
[2021-12-22 01:42:19,603] {processor.py:163} INFO - Started process (PID=123) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:42:19,605] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:42:19,606] {logging_mixin.py:109} INFO - [2021-12-22 01:42:19,606] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:42:19,618] {logging_mixin.py:109} INFO - [2021-12-22 01:42:19,616] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 42, in <module>
    bash_command='python /opt/airflow/resoures/test.py'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 143, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 547, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (call python sript) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:42:19,638] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:42:19,648] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.050 seconds
[2021-12-22 01:42:49,763] {processor.py:163} INFO - Started process (PID=149) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:42:49,766] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:42:49,768] {logging_mixin.py:109} INFO - [2021-12-22 01:42:49,768] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:42:49,785] {logging_mixin.py:109} INFO - [2021-12-22 01:42:49,781] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 42, in <module>
    bash_command='python /opt/airflow/resoures/test.py'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 143, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 547, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (call python sript) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:42:49,805] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:42:49,813] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.054 seconds
[2021-12-22 01:43:19,948] {processor.py:163} INFO - Started process (PID=185) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:43:19,950] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:43:19,951] {logging_mixin.py:109} INFO - [2021-12-22 01:43:19,951] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:43:19,970] {logging_mixin.py:109} INFO - [2021-12-22 01:43:19,967] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 42, in <module>
    bash_command='python /opt/airflow/resoures/test.py'
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 143, in __init__
    super().__init__(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 188, in apply_defaults
    result = func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 547, in __init__
    validate_key(task_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (call python sript) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:43:19,991] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:43:20,004] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.060 seconds
[2021-12-22 01:43:20,966] {processor.py:163} INFO - Started process (PID=186) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:43:20,968] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:43:20,969] {logging_mixin.py:109} INFO - [2021-12-22 01:43:20,969] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:43:20,989] {logging_mixin.py:109} INFO - [2021-12-22 01:43:20,985] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 37, in <module>
    tags=['example'],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 375, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (Example 2) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:43:21,011] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:43:21,020] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.059 seconds
[2021-12-22 01:43:23,042] {processor.py:163} INFO - Started process (PID=187) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:43:23,044] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:43:23,046] {logging_mixin.py:109} INFO - [2021-12-22 01:43:23,046] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:43:23,057] {logging_mixin.py:109} INFO - [2021-12-22 01:43:23,056] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 37
    tags=['example' 2],
                    ^
SyntaxError: invalid syntax
[2021-12-22 01:43:23,083] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:43:23,095] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.056 seconds
[2021-12-22 01:43:24,057] {processor.py:163} INFO - Started process (PID=188) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:43:24,059] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:43:24,060] {logging_mixin.py:109} INFO - [2021-12-22 01:43:24,060] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:43:24,077] {logging_mixin.py:109} INFO - [2021-12-22 01:43:24,075] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 37, in <module>
    tags=['example 2'],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 375, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (Example 2) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:43:24,095] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:43:24,112] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.059 seconds
[2021-12-22 01:43:32,879] {processor.py:163} INFO - Started process (PID=204) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:43:32,882] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:43:32,885] {logging_mixin.py:109} INFO - [2021-12-22 01:43:32,884] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:43:32,906] {logging_mixin.py:109} INFO - [2021-12-22 01:43:32,903] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 37, in <module>
    tags=['example 2'],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 375, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (Example 2) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:43:32,925] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:43:32,933] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.058 seconds
[2021-12-22 01:43:52,946] {processor.py:163} INFO - Started process (PID=215) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:43:52,949] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:43:52,951] {logging_mixin.py:109} INFO - [2021-12-22 01:43:52,951] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:43:52,976] {logging_mixin.py:109} INFO - [2021-12-22 01:43:52,973] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 37, in <module>
    tags=['example 2'],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 375, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (Example 2) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:43:53,003] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:43:53,012] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.070 seconds
[2021-12-22 01:43:59,721] {processor.py:163} INFO - Started process (PID=231) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:43:59,723] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:43:59,724] {logging_mixin.py:109} INFO - [2021-12-22 01:43:59,724] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:43:59,748] {logging_mixin.py:109} INFO - [2021-12-22 01:43:59,745] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 37, in <module>
    tags=['example 2'],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 375, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (Example 2) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:43:59,766] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:43:59,774] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.058 seconds
[2021-12-22 01:44:00,737] {processor.py:163} INFO - Started process (PID=232) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:44:00,739] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:44:00,741] {logging_mixin.py:109} INFO - [2021-12-22 01:44:00,741] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:44:00,762] {logging_mixin.py:109} INFO - [2021-12-22 01:44:00,759] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 37, in <module>
    tags=['example 2'],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 375, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (Example 2) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:44:00,781] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:44:00,790] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.057 seconds
[2021-12-22 01:44:30,903] {processor.py:163} INFO - Started process (PID=256) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:44:30,905] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:44:30,906] {logging_mixin.py:109} INFO - [2021-12-22 01:44:30,906] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:44:30,920] {logging_mixin.py:109} INFO - [2021-12-22 01:44:30,917] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/example2.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example2.py", line 37, in <module>
    tags=['example 2'],
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dag.py", line 375, in __init__
    validate_key(dag_id)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/helpers.py", line 54, in validate_key
    "dots and underscores exclusively".format(k=k)
airflow.exceptions.AirflowException: The key (Example 2) has to be made of alphanumeric characters, dashes, dots and underscores exclusively
[2021-12-22 01:44:30,938] {processor.py:639} WARNING - No viable dags retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:44:30,948] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.049 seconds
[2021-12-22 01:44:42,985] {processor.py:163} INFO - Started process (PID=267) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:44:42,987] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:44:42,989] {logging_mixin.py:109} INFO - [2021-12-22 01:44:42,988] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:44:43,046] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:44:46,427] {logging_mixin.py:109} INFO - [2021-12-22 01:44:46,427] {providers_manager.py:156} WARNING - Exception when importing 'airflow.providers.google.leveldb.hooks.leveldb.LevelDBHook' from 'apache-airflow-providers-google' package: No module named 'plyvel'
[2021-12-22 01:44:48,213] {logging_mixin.py:109} INFO - [2021-12-22 01:44:48,213] {providers_manager.py:156} WARNING - Exception when importing 'airflow.providers.google.leveldb.hooks.leveldb.LevelDBHook' from 'apache-airflow-providers-google' package: No module named 'plyvel'
[2021-12-22 01:44:48,239] {logging_mixin.py:109} INFO - [2021-12-22 01:44:48,239] {manager.py:496} INFO - Created Permission View: can read on DAG:Example_2
[2021-12-22 01:44:48,248] {logging_mixin.py:109} INFO - [2021-12-22 01:44:48,248] {manager.py:496} INFO - Created Permission View: can edit on DAG:Example_2
[2021-12-22 01:44:48,249] {logging_mixin.py:109} INFO - [2021-12-22 01:44:48,249] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:44:48,257] {logging_mixin.py:109} INFO - [2021-12-22 01:44:48,257] {dag.py:2415} INFO - Creating ORM DAG for Example_2
[2021-12-22 01:44:48,267] {logging_mixin.py:109} INFO - [2021-12-22 01:44:48,267] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-21T01:44:48.267501+00:00
[2021-12-22 01:44:48,284] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 5.325 seconds
[2021-12-22 01:44:56,035] {processor.py:163} INFO - Started process (PID=281) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:44:56,037] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:44:56,040] {logging_mixin.py:109} INFO - [2021-12-22 01:44:56,040] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:44:56,109] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:44:56,125] {logging_mixin.py:109} INFO - [2021-12-22 01:44:56,125] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:44:56,149] {logging_mixin.py:109} INFO - [2021-12-22 01:44:56,149] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-21T01:44:56.148915+00:00
[2021-12-22 01:44:56,168] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.138 seconds
[2021-12-22 01:45:26,297] {processor.py:163} INFO - Started process (PID=323) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:45:26,300] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:45:26,301] {logging_mixin.py:109} INFO - [2021-12-22 01:45:26,301] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:45:26,346] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:45:26,743] {logging_mixin.py:109} INFO - [2021-12-22 01:45:26,743] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:45:26,767] {logging_mixin.py:109} INFO - [2021-12-22 01:45:26,766] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:45:26,785] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.498 seconds
[2021-12-22 01:45:56,949] {processor.py:163} INFO - Started process (PID=348) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:45:56,951] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:45:56,952] {logging_mixin.py:109} INFO - [2021-12-22 01:45:56,952] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:45:56,995] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:45:57,010] {logging_mixin.py:109} INFO - [2021-12-22 01:45:57,010] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:45:57,031] {logging_mixin.py:109} INFO - [2021-12-22 01:45:57,031] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:45:57,047] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.103 seconds
[2021-12-22 01:46:04,851] {processor.py:163} INFO - Started process (PID=364) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:46:04,854] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:46:04,856] {logging_mixin.py:109} INFO - [2021-12-22 01:46:04,856] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:46:04,905] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:46:05,279] {logging_mixin.py:109} INFO - [2021-12-22 01:46:05,279] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:46:05,295] {logging_mixin.py:109} INFO - [2021-12-22 01:46:05,295] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:46:05,311] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.465 seconds
[2021-12-22 01:46:35,433] {processor.py:163} INFO - Started process (PID=390) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:46:35,435] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:46:35,440] {logging_mixin.py:109} INFO - [2021-12-22 01:46:35,440] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:46:35,491] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:46:35,514] {logging_mixin.py:109} INFO - [2021-12-22 01:46:35,514] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:46:35,545] {logging_mixin.py:109} INFO - [2021-12-22 01:46:35,545] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:46:35,562] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.140 seconds
[2021-12-22 01:47:05,663] {processor.py:163} INFO - Started process (PID=416) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:47:05,665] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:47:05,666] {logging_mixin.py:109} INFO - [2021-12-22 01:47:05,666] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:47:05,701] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:47:05,713] {logging_mixin.py:109} INFO - [2021-12-22 01:47:05,713] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:47:05,732] {logging_mixin.py:109} INFO - [2021-12-22 01:47:05,732] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:47:05,744] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.085 seconds
[2021-12-22 01:47:35,814] {processor.py:163} INFO - Started process (PID=453) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:47:35,816] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:47:35,817] {logging_mixin.py:109} INFO - [2021-12-22 01:47:35,817] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:47:35,853] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:47:35,865] {logging_mixin.py:109} INFO - [2021-12-22 01:47:35,865] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:47:35,886] {logging_mixin.py:109} INFO - [2021-12-22 01:47:35,886] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:47:35,899] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.089 seconds
[2021-12-22 01:48:05,966] {processor.py:163} INFO - Started process (PID=477) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:48:05,968] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:48:05,969] {logging_mixin.py:109} INFO - [2021-12-22 01:48:05,969] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:48:06,009] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:48:06,026] {logging_mixin.py:109} INFO - [2021-12-22 01:48:06,025] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:48:06,048] {logging_mixin.py:109} INFO - [2021-12-22 01:48:06,048] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:48:06,067] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.105 seconds
[2021-12-22 01:49:18,813] {processor.py:163} INFO - Started process (PID=40) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:49:18,848] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:49:18,853] {logging_mixin.py:109} INFO - [2021-12-22 01:49:18,852] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:49:19,088] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:49:19,166] {logging_mixin.py:109} INFO - [2021-12-22 01:49:19,166] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:49:19,222] {logging_mixin.py:109} INFO - [2021-12-22 01:49:19,222] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:49:19,242] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.440 seconds
[2021-12-22 01:49:50,074] {processor.py:163} INFO - Started process (PID=65) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:49:50,076] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:49:50,077] {logging_mixin.py:109} INFO - [2021-12-22 01:49:50,077] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:49:50,131] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:49:50,160] {logging_mixin.py:109} INFO - [2021-12-22 01:49:50,160] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:49:50,190] {logging_mixin.py:109} INFO - [2021-12-22 01:49:50,189] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:49:50,209] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.139 seconds
[2021-12-22 01:50:20,737] {processor.py:163} INFO - Started process (PID=105) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:50:20,739] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:50:20,741] {logging_mixin.py:109} INFO - [2021-12-22 01:50:20,740] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:50:20,782] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:50:20,794] {logging_mixin.py:109} INFO - [2021-12-22 01:50:20,794] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:50:20,814] {logging_mixin.py:109} INFO - [2021-12-22 01:50:20,814] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:50:20,828] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.095 seconds
[2021-12-22 01:50:50,937] {processor.py:163} INFO - Started process (PID=140) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:50:50,939] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:50:50,941] {logging_mixin.py:109} INFO - [2021-12-22 01:50:50,941] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:50:50,996] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:50:51,008] {logging_mixin.py:109} INFO - [2021-12-22 01:50:51,008] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:50:51,028] {logging_mixin.py:109} INFO - [2021-12-22 01:50:51,028] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:50:51,037] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.107 seconds
[2021-12-22 01:51:21,120] {processor.py:163} INFO - Started process (PID=169) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:51:21,122] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:51:21,124] {logging_mixin.py:109} INFO - [2021-12-22 01:51:21,123] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:51:21,163] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:51:21,183] {logging_mixin.py:109} INFO - [2021-12-22 01:51:21,182] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:51:21,205] {logging_mixin.py:109} INFO - [2021-12-22 01:51:21,205] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:51:21,217] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.102 seconds
[2021-12-22 01:51:51,317] {processor.py:163} INFO - Started process (PID=195) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:51:51,319] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:51:51,321] {logging_mixin.py:109} INFO - [2021-12-22 01:51:51,320] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:51:51,355] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:51:51,367] {logging_mixin.py:109} INFO - [2021-12-22 01:51:51,367] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:51:51,385] {logging_mixin.py:109} INFO - [2021-12-22 01:51:51,385] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:51:51,398] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.085 seconds
[2021-12-22 01:52:21,483] {processor.py:163} INFO - Started process (PID=234) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:52:21,485] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:52:21,486] {logging_mixin.py:109} INFO - [2021-12-22 01:52:21,486] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:52:21,521] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:52:21,534] {logging_mixin.py:109} INFO - [2021-12-22 01:52:21,534] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:52:21,553] {logging_mixin.py:109} INFO - [2021-12-22 01:52:21,553] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:52:21,566] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.088 seconds
[2021-12-22 01:52:51,659] {processor.py:163} INFO - Started process (PID=259) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:52:51,661] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:52:51,662] {logging_mixin.py:109} INFO - [2021-12-22 01:52:51,662] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:52:51,698] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:52:51,712] {logging_mixin.py:109} INFO - [2021-12-22 01:52:51,712] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:52:51,731] {logging_mixin.py:109} INFO - [2021-12-22 01:52:51,731] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:52:51,746] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.091 seconds
[2021-12-22 01:53:21,821] {processor.py:163} INFO - Started process (PID=294) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:53:21,823] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:53:21,825] {logging_mixin.py:109} INFO - [2021-12-22 01:53:21,825] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:53:21,870] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:53:21,886] {logging_mixin.py:109} INFO - [2021-12-22 01:53:21,886] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:53:21,909] {logging_mixin.py:109} INFO - [2021-12-22 01:53:21,909] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:53:21,926] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.111 seconds
[2021-12-22 01:53:52,014] {processor.py:163} INFO - Started process (PID=319) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:53:52,016] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:53:52,018] {logging_mixin.py:109} INFO - [2021-12-22 01:53:52,018] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:53:52,067] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:53:52,081] {logging_mixin.py:109} INFO - [2021-12-22 01:53:52,081] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:53:52,111] {logging_mixin.py:109} INFO - [2021-12-22 01:53:52,111] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:53:52,126] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.122 seconds
[2021-12-22 01:54:22,182] {processor.py:163} INFO - Started process (PID=345) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:54:22,184] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:54:22,186] {logging_mixin.py:109} INFO - [2021-12-22 01:54:22,186] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:54:22,226] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:54:22,239] {logging_mixin.py:109} INFO - [2021-12-22 01:54:22,239] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:54:22,266] {logging_mixin.py:109} INFO - [2021-12-22 01:54:22,266] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:54:22,280] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.102 seconds
[2021-12-22 01:54:52,418] {processor.py:163} INFO - Started process (PID=380) to work on /opt/airflow/dags/example2.py
[2021-12-22 01:54:52,421] {processor.py:625} INFO - Processing file /opt/airflow/dags/example2.py for tasks to queue
[2021-12-22 01:54:52,424] {logging_mixin.py:109} INFO - [2021-12-22 01:54:52,424] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/example2.py
[2021-12-22 01:54:52,468] {processor.py:637} INFO - DAG(s) dict_keys(['Example_2']) retrieved from /opt/airflow/dags/example2.py
[2021-12-22 01:54:52,491] {logging_mixin.py:109} INFO - [2021-12-22 01:54:52,491] {dag.py:2396} INFO - Sync 1 DAGs
[2021-12-22 01:54:52,518] {logging_mixin.py:109} INFO - [2021-12-22 01:54:52,518] {dag.py:2928} INFO - Setting next_dagrun for Example_2 to 2021-12-22T01:44:56.148915+00:00
[2021-12-22 01:54:52,534] {processor.py:171} INFO - Processing /opt/airflow/dags/example2.py took 0.124 seconds
